react/src/components/CreateMapModal/CreateMapModal.module.css

```diff
@@ -11,10 +11,54 @@
 
 .custom-error-message {
   margin-top: 0.25rem;
-  font-size: 0.8em;
+  font-size: 1.2em;
   color: #dc3545;
 }
 
 .form-check-label {
   font-style: italic;
 }
```

Comment:
**Outdated classname? Consider deleting it.**

When I search this page, I only see this class name here, and where it was deleted.